'

```diff
@@ -0,0 +1,32 @@
+;; Generic tests on sumtypes.
```

Comment:
Are those tests pulled in by @scolsen’s commit?

Reply:
Yeah, that was a goof on my part that I removed. (I somehow copied and saved that test file as `'`).

I removed that in another commit, so pulling in/merging master to this branch should fix this.

Reply:
File removed.